packages/expo-face-detector/src/FaceDetector.ts

```diff
@@ -185,6 +187,13 @@ export type DetectionOptions = {
   tracking?: boolean;
 };
 
+if (!warnedAboutDeprecation) {
+  console.warn(
+    'ExpoFaceDetector has been deprecated and will be removed in a future SDK version. We recommend using react-native-vision-camera for this functionality.'
```

Comment:
```suggestion
    'ExpoFaceDetector has been deprecated and will be removed in a future SDK version. We recommend using react-native-vision-camera for this functionality. See https://github.com/mrousavy/react-native-vision-camera'
```

maybe find a way to show the URL here?